apis/release/v1alpha1/types.go

```diff
@@ -60,6 +60,7 @@ type SetVal struct {
 
 // ValuesSpec defines the Helm value overrides spec for a Release
 type ValuesSpec struct {
+	// +kubebuilder:pruning:PreserveUnknownFields
```

Comment:
Should this be on the overall `Release` struct with the other `kubebuilder` annotations?

Reply:
As I understood it, with v1 CRDs this is a field level configuration.

Reply:
From the docs:
> By default, all unspecified fields for a custom resource, across all versions, are pruned. It is possible though to opt-out of that for specifc sub-trees of fields by adding x-kubernetes-preserve-unknown-fields: true in the structural OpenAPI v3 validation schema.

https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#controlling-pruning

Reply:
ah nice, looks like you are correct :+1:

Reply:
We actually hit this same issue in c/c but it was caught before release https://github.com/crossplane/crossplane/pull/1858